bin/do_Pk1D.py

```diff
@@ -131,10 +133,21 @@ def compute_mean_delta(ll,delta,iv,zqso):
         tree = TTree("Pk1D","SDSS 1D Power spectrum Ly-a");
         zqso,mean_z,mean_reso,mean_SNR,lambda_min,lambda_max,plate,mjd,fiber,\
         nb_mask_pix,nb_r,k_r,Pk_r,Pk_raw_r,Pk_noise_r,cor_reso_r,Pk_diff_r = make_tree(tree,nb_bin_max)
-        hdelta  = TProfile2D( 'hdelta', 'delta mean as a function of lambda-lambdaRF', 36, 3600., 7200., 16, 1040., 1200., -5.0, 5.0)
-        hdelta_RF  = TProfile( 'hdelta_RF', 'delta mean as a function of lambdaRF', 320, 1040., 1200., -5.0, 5.0)
+
+        # control histograms
+        if (args.forest_type=='Lya'):
+            forest_inf=1040.
+            forest_sup=1200.
+        elif (args.forest_type=='SiIV'):
+            forest_inf=1270.
+            forest_sup=1380.
+        elif (args.forest_type=='CIV'):
+            forest_inf=1410.
+            forest_sup=1520.
```

Comment:
why not read forest_inf and forest_sup from parser.parse_args()?

Reply:
It is just for the histograms, by definition it is the maximal width of the forest, I want that the limits are fixed. By this way I can see if in do_deltas I have used a narrower forest